dll/win32/shell32/shlexec.cpp

```diff
@@ -1569,20 +1569,20 @@ static HRESULT ShellExecute_ContextMenuVerb(LPSHELLEXECUTEINFOW sei)
     ici.hwnd = sei->hwnd;
     ici.lpParameters = parameters;
     ici.lpParametersW = sei->lpParameters;
-    if ((sei->fMask & (SEE_MASK_HASLINKNAME | SEE_MASK_CLASSNAME)) == SEE_MASK_HASLINKNAME)
-    {
-        ici.fMask |= CMIC_MASK_HASLINKNAME;
+    ici.dwHotKey = sei->dwHotKey;
+    ici.hIcon = sei->hIcon;
+    if (ici.fMask & (CMIC_MASK_HASLINKNAME | CMIC_MASK_HASTITLE))
```

Comment:
Is it actually correct to set the `ici.lpTitleW` to `sei->lpClass` even if the `fMask` (of `ici` or `sei`) has only `CMIC_MASK_HASTITLE` but not `CMIC_MASK_HASLINKNAME` ?
`sei->lpClass` is invalid if `sei->fMask` doesn't have `SEE_MASK_CLASSNAME`; but here no check for `SEE_MASK_CLASSNAME` is made anymore...

Reply:
Ah ok wait, this commit https://github.com/reactos/reactos/commit/471be3e8680e9ead53effa44fab34893398b4d45 appears to explain the reason.